src/main/java/pro/sky/course2CourseWork/controller/JavaController.java

```diff
@@ -0,0 +1,35 @@
+package pro.sky.course2CourseWork.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pro.sky.course2CourseWork.domain.Question;
+import pro.sky.course2CourseWork.service.QuestionService;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/exam")
```

Comment:
```suggestion
@RequestMapping("/exam/java")
```
чтобы не дублировать далее по методам